### DISCUSSION TITLE
How to play videos in an App

### DISCUSSION THREAD

**Discussion by heromiyo** [mediacms-io/mediacms#270] [Q&A]
I uploaded a video via api and would like to play in an app, the url in the response cant play in app

**Comment by mgogoulos** [comment]
Hi, you get back the API url (that contains info regarding the media files) and the media URL, I'm assuming what you need is the embed URL? For example https://demo.mediacms.io/embed?m=TrCcdrxk5 (note that if you replace embed with view, you get the media page)
